app/models/fraud/indicator.rb

```diff
@@ -73,7 +73,8 @@ def duplicates(references)
       # skip this rule if we can't check against the reference object
       return passing_response if references[reference].blank?
 
-      duplicate_ids = DeduplificationService.duplicates(references[reference], *indicator_attributes, from_scope: query_model_name.constantize).pluck(:id)
+      from_scope = query_model.respond_to?(:accessible_intakes) ? query_model.accessible_intakes : query_model
```

Comment:
for duplicate checks on intakes, we want to narrow things down to only accessible_intakes, or else we'll be overcounting duplicates by including intakes where the client entered a phone number or email but never verified it, for example.